javascripts/discourse/components/kanban/wrapper.gjs

```diff
@@ -11,6 +11,35 @@ import KanbanOptionsModal from "./modal/options";
 import { getOwner } from "@ember/application";
 
 export default class Kanban extends Component {
+  @service kanbanManager;
+  @service modal;
+
+  @tracked dragData;
+
+  get isLegacyTopicList() {
```

Comment:
I think we can drop this now. The core topic list refactor was merged a while back, so discourse-compatibility should be all we need.

Should be able to remove the `.kanban-legacy-topic-list` css as well